### DISCUSSION TITLE
Is there any way to enable AMD overclocking menu in Lenovo Legion 5 laptop?

### DISCUSSION THREAD

**Discussion by kolebynov** [DavidS95/Smokeless_UMAF#3] [Q&A] [answered]
Hi all,
AMD Overclocking menu isn't present in UniversalAMDFormBrowser for Lenovo Legion 5 laptop. My BIOS is GKCN59WW. Is there any way to enable this menu in UniversalAMDFormBrowser?

**Comment by DavidS95** [comment, marked as answer]
The amd overclocking menu only shows up on Ryzen 9 HX/HS cpus that are unlocked, the tool currently only reveals hidden bios menus if the menu isnt there then it wont show up.

**Reply by kolebynov** [reply]
I got it, thanks for the answer
